src/components/tooltip/wrapper.js

```diff
@@ -0,0 +1,5 @@
+import styled from '@emotion/styled';
+
+export default styled.div`
```

Comment:
Nitpick: we can define this component in the `tooltip.js` file directly I suppose, right?